test/parallel/test-event-emitter-check-listener-leaks.js

```diff
@@ -13,6 +13,14 @@ assert.ok(!e._events['default'].hasOwnProperty('warned'));
 e.on('default', function() {});
 assert.ok(e._events['default'].warned);
 
+// symbol
+var sym = Symbol('x');
```

Comment:
If you use `const` and `let`, you can also isolate the tests using curly braces. So:

```js
{
  const sym = Symbol('x');

  e.setMaxListeners(1);
  ...
}
```